NitroxInstallerActions/Patches/FirewallRules.cs

```diff
@@ -46,112 +27,146 @@ string RunNetshCommand(string Argument)
             netshProcess.Close();
             return output;
         }
-        bool SetFirewallRules(List<string> RulesArray, string SubPath, int PortNumber)
+        public string addCommand;
+        public string removeCommand;
+        public string name;
+        public bool isInstalled;
+        string[] GetAddRemoveCommand(string RuleName, string GamePath, int Port = 11000)
         {
-            bool status = true;
-            foreach(string item in RulesArray)
+            string[] commands = new string[2] { "", "" };
+            if (RuleName == "PortsUDP")
```

Comment:
Use an enum for RuleName instead of string. Less sensitive to errors and easier to check.